zstacklib/zstacklib/utils/lvm.py

```
@@ -2743,3 +2764,361 @@ def check_lvmlockd_log():
+    gllk_offset = sanlock.GLLK_BEGIN * align
+    sanlock.direct_init_resource(
+        "%s:GLLK:%s:%d" % (lockspace_name, dm_path, gllk_offset))
+
+    # 3. VGLK at offset 66 * align.
+    vglk_offset = sanlock.VGLK_BEGIN * align
+    sanlock.direct_init_resource(
+        "%s:VGLK:%s:%d" % (lockspace_name, dm_path, vglk_offset))
+
+    # 4. Per-LV resource leases.
+    #    Each LV has lock_args "1.0.0:<slot>", slot * align = offset.
+    r, o, e = bash.bash_roe(
```

Ruff (0.15.6)

[warning] Unpacked variable `e` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)